autoload/pandoc/keyboard/styles.vim

```diff
@@ -95,10 +98,18 @@ function! pandoc#keyboard#styles#ToggleOperator(type, ends)
     let match_data = matchlist(getreg(reg), '\('.s:EscapeEnds(a:ends).'\)\(.*\)\('.s:EscapeEnds(a:ends).'\)')
     if len(match_data) == 0
         call setreg(reg, a:ends.getreg(reg).a:ends)
-        execute 'normal "'.reg.'P'
+        if s:IsCursorAtEndofNonEmptyLine()
```

Comment:
This could be simplified to this, inline:
~~~
if (ccol+1) == col('$') && ccol) != 1
~~~
but it doesn't make much difference.